ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java

```diff
@@ -300,4 +348,204 @@ public void stop() {
     private BurnupChart getNewBurnupChart() {
         return new BurnupChart(SPRINT_LENGTH_IN_DAYS, false);
     }
+    
+    /* This method is used to build the Stage for editing the team attributes. */
+    private void buildEditTeamStage() {
+    	Stage editTeamStage = new Stage();
+    	editTeamStage.setTitle("Edit Team");
+    	
+    	Label teamLabel = new Label();
+    	teamLabel.textProperty().bind(team.nameProperty());
+    	teamLabel.setFont(Font.font("Lucida Sans", FontWeight.BOLD, 20.0));
+    	teamLabel.setPadding(new Insets(0.0, 0.0, 0.0, 20.0));
+    	teamLabel.setTextFill(SCRUM_BOARD_BLUE);
+    	
+    	Label velocityLabel = new Label("Velocity:");
+    	TextField velocityField = new TextField();
+    	velocityField.setPrefWidth(60.0);
+    	Label errorLabel = new Label("Input must be an integer!");
+    	errorLabel.setTextFill(Color.RED);
+    	errorLabel.setVisible(false);
+    	HBox velocityPane = new HBox(10.0);
+    	velocityPane.getChildren().addAll(velocityLabel, velocityField, errorLabel);
+    	
+    	Button cancelButton = new Button("Cancel");
+    	cancelButton.setPrefWidth(100.0);
+    	cancelButton.setOnAction((event) -> editTeamStage.close());
+    	
+    	Button acceptButton = new Button("Accept");
+    	acceptButton.setPrefWidth(100.0);
+    	acceptButton.setOnAction((event) -> {
+    		try {
+    			team.setVelocity(Integer.parseInt(velocityField.getText()));
+    			editTeamStage.close();
+    		} catch (NumberFormatException e) {
+    			errorLabel.setVisible(true);
+    		}
+    	});
+    	
+    	HBox buttonPane = new HBox(10.0);
+    	buttonPane.getChildren().addAll(cancelButton, acceptButton);
+    	
+    	VBox editTeamPane = new VBox(10.0);
+    	editTeamPane.setPadding(new Insets(20.0));
+    	editTeamPane.getChildren().addAll(teamLabel, velocityPane, buttonPane);
+    	
+    	editTeamStage.setScene(new Scene(editTeamPane, 350.0, 300.0));
+    	editTeamStage.show();
+    }
+    
+    public void buildBurndownChartStage() {
+
+        Stage burndownChartStage = new Stage();
+        burndownChartStage.setTitle("Burndown Chart");
+
+        Label burndownLabel = new Label("Burndown Chart");
+        VBox burndownChartPane = new VBox(10.0);
+        burndownChartPane.setPadding(new Insets(20.0));
+        burndownChartPane.getChildren().addAll(burndownLabel);
+
+        final NumberAxis xAxis = new NumberAxis(0, 10, 1);
+        final NumberAxis yAxis = new NumberAxis(0,26,1);
+
+
+        AreaChart burnDownChart = new AreaChart(xAxis, yAxis);
+
+
+        BorderPane borderpane = new BorderPane(burndownLabel);
+        borderpane.setAlignment(burndownLabel, Pos.TOP_CENTER);
+
+        borderpane.setCenter(burnDownChart);
+
+
+        //burndownChartStage.setScene(scene);
+        burndownChartStage.show();
+
+    }
+    
+    /* This method is used to build the Stage for editing the team attributes. */
+    private void buildTaigaIntegrationStage() {
```

Comment:
Please move this to a separate file for modularity. Additionally this makes the file super long, as well as it's different then the rest of the code that does a similar thing. Also there's an option for import from taiga in the file menu. I believe the pop out should be linked to that.